tests/dummy/config/environment.js

```diff
@@ -43,6 +43,8 @@ module.exports = function(environment) {
 
     apiHost: 'http://localhost:4200',
 
+    googleClientID: '271351926386-jdh86lc996s5r6kia2dk2r7edrsmke85.apps.googleusercontent.com',
```

Comment:
Yeah it's a very confusing console IMHO. The URL for the credentials section is: `https://console.developers.google.com/apis/credentials?project=<project-name>` and then in the `OAuth 2.0 client IDs` table, find the right row, then click the Edit/pencil icon, scroll down to `Authorized redirect URIs` and add it there. Hope that works!